clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -167,6 +167,12 @@ New checks
   Detects default initialization (to 0) of variables with ``enum`` type where
   the enum has no enumerator with value of 0.
 
+- New :doc:`bugprone-loop-variable-copied-then-modified
+  <clang-tidy/checks/bugprone/loop-variable-copied-then-modified>` check.
+
+  Detects when a loop variable is copied and then subsequently (possibly) modified
+  and suggests replacing with a reference or an explicit copy.
+
 - New :doc:`bugprone-derived-method-shadowing-base-method
```

Comment:
Please keep alphabetical order (by check name) in this section.

Reply:
alphabetized the checks~